src/client/src/mockData/mockVsCodeApi.ts

```diff
@@ -84,7 +84,20 @@ const mockVsCodeApi = () => ({
                 locations: [{ label: WEST_US, value: WEST_US }],
                 resourceGroups: [
                   { label: RESOURCE_GROUP_MOCK, value: RESOURCE_GROUP_MOCK },
-                  { label: "ResourceGroupMock2", value: "ResourceGroupMock2" }
+                  { label: "ResourceGroupMock2", value: "ResourceGroupMock2" },
+                  { label: "ResourceGroupMock2", value: "ResourceGroupMock2" },
+                  { label: "ResourceGroupMock3", value: "ResourceGroupMock3" },
+                  { label: "ResourceGroupMock4", value: "ResourceGroupMock4" },
+                  { label: "ResourceGroupMock5", value: "ResourceGroupMock5" },
+                  { label: "ResourceGroupMock6", value: "ResourceGroupMock6" },
+                  { label: "ResourceGroupMock7", value: "ResourceGroupMock7" },
+                  { label: "ResourceGroupMock8", value: "ResourceGroupMock8" },
+                  { label: "ResourceGroupMock9", value: "ResourceGroupMock9" },
+                  {
+                    label: "ResourceGroupMock10",
```

Comment:
Format does not look consistent with the rest

Reply:
Yea, this was done by Prettier not me lol

Reply:
Maybe I should stop using Prettier everywhere

Reply:
No, you should have consistent settings and use it everywhere. Otherwise we will have a lot of noise (formatting changes) in PRs